style.css

```diff
@@ -147,3 +157,71 @@ html {
 .card-overlay:hover {
     background-color: var(--COLOR-SHADOW-3);
 }
+
+/* GRID FOR PRODUCT DISPLAY */
+
+.grid-products {
+    display: grid;
+    grid-template-columns: 25rem 1fr;
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+        "navbar navbar"
+        "sidebar main"
+        "footer footer";
+}
+
+/* PRODUCT LIST DISPLAY */
+
+.products-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: start;
+    align-items: center;
+    gap: 2rem;
+    margin: 3rem 0;
+}
+
+/* PRODUCT FILTER */
+
+.grand-filter {
+    background-color: var(--WHITE);
+    padding: 0 2rem;
+}
+
+.grand-filter-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+}
+
+/* CUSTOM SLIDER */
+
+.grand-slider-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+.slider-range {
+    display: flex;
+    justify-content: space-between;
+    margin: 1rem 0;
+}
+
+.grand-slider {
+    display: flex;
+    align-items: center;
+    appearance: none;
+    background-color: var(--PRIMARY-50);
+    height: 8px;
+    border-radius: 99px;
```

Comment:
use rem here

Reply:
Thank you srishti for pointing this. Made required changes